app/views/devise/sessions/new.html.erb

```diff
@@ -37,6 +37,7 @@
         <li>If you don't have a membership with us yet, <%= link_to "sign up online", signup_path %>.</li>
       <% end %>
         <li>If you need to choose or don't remember your password, <%= link_to "reset your password", new_user_password_path %>.</li>
+        <li>If you need haven't confirmed your email address, <%= link_to "request a confirmation email", new_user_confirmation_path %>.</li>
```

Comment:
```suggestion
        <li>If you haven't confirmed your email address, <%= link_to "request a confirmation email", new_user_confirmation_path %>.</li>
```

Found a typo, though I think it might be more confusing to have this showing to every logging in user vs only showing it in flash messages when the email is not verified.

Reply:
It looks like we can modify the devise settings so the flash message includes a link instead. https://stackoverflow.com/questions/4008059/rails-devise-customize-flash-message-devise-en-yml-with-a-link-to